haxepunk/Entity.hx

```diff
@@ -945,8 +943,5 @@ class Entity extends Tweener
 	var _moveX:Float = 0;
 	var _moveY:Float = 0;
 
-	// Rendering information.
-	var _point:Point;
```

Comment:
I was running into issues after this change due to nested calls modifying HXP.point. Seems reasonable for Entity etc. to keep its own static point to use when rendering.